trial-config-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+spring.liquibase.change-log=classpath:/db/changelog/changelog-master.xml
+spring.liquibase.drop-first=false
+spring.liquibase.enabled=true
+spring.jpa.hibernate.ddl-auto=none
+
+spring.hateoas.use-hal-as-default-json-media-type=false
+
+
+# Log properties in application.properties
+#logging.level.org.springframework.security=DEBUG
+
+# Specifies your Active Directory ID:
+spring.security.oauth2.resourceserver.jwt.issuer-uri=https://sts.windows.net/99804659-431f-48fa-84c1-65c9609de05b/
+spring.security.oauth2.resourceserver.opaquetoken.client-id=901b377a-a4c0-4a72-b390-294cef5c0c74
+spring.security.oauth2.resourceserver.opaquetoken.client-secret=wq99fHb3Q9Vg_LfsZ_pNEG6~XR_h8um6u4
```

Comment:
This secret is now compromised. Please remove from the code and create a new secret.
It's better to create a managed identity instead of committing secrets into a public repo.

Reply:
should these be best placed in the key vault ?